input/postgres/superuser.go

```diff
@@ -100,3 +100,21 @@ func StatsHelperExists(ctx context.Context, db *sql.DB, statsHelper string) bool
 
 	return enabled
 }
+
+const statsHelperReturnTypeSQL string = `
+SELECT pg_catalog.format_type(prorettype, null)
```

Comment:
This returns `pg_stats` for the old function, and `record` for the new function.